.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -5,19 +5,95 @@ description: Suggest an idea for this project
+      small sub group should be [implemented in a third-party Ansible
+      Collection][contribute to collections] or maybe even just your
+      project alone. Be mindful of the fact that the essential
+      ansible-core features have a broad impact.
+
+      If unsure, consider filing a [new proposal] instead outlining your
+      use-cases, the research and implementation considerations. Then,
+      start a discussion on one of the public [IRC meetings] we have just
+      for this.
+
+
+      <details>
+        <summary>
+          ❗ Every change breaks someone's workflow.
+        </summary>
+
+
+        [![❗ Every change breaks someone's workflow.
+        ](https://imgs.xkcd.com/comics/workflow.png)
+        ](https://xkcd.com/1172/)
+      </details>
+
+
       ⚠
-      Verify first that your issue is not [already reported on GitHub][issue search].
-      Also test if the latest release and devel branch are affected too.
-      *Complete **all** sections as described, this form is processed automatically.*
+      Verify first that your idea is not [already requested on
+      GitHub][issue search].
+
+      Also test if the devel branch does not already implement this.
+
+      *Complete **all** sections as described, this form is processed
+      automatically.*
+
+
+      **Tip:** If you are seeking community support, please consider
+      [starting a mailing list thread or chatting in IRC][ML||IRC].
+
+
+      Thank you for your collaboration!
+
+
 
-      [issue search]: https://github.com/ansible/ansible/search?q=is%3Aissue&type=issues
+      [contribute to collections]:
+      https://docs.ansible.com/ansible-core/devel/community/contributing_maintained_collections.html?utm_medium=github&utm_source=issue_form--feature_request.yml
+
+      [IRC meetings]:
+        https://docs.ansible.com/ansible-core/devel/community/communication.html?utm_medium=github&utm_source=issue_form--feature_request.yml#irc-meetings
+
+      [issue search]: ../search?q=is%3Aissue&type=issues
+
+      [ML||IRC]:
+      https://docs.ansible.com/ansible-core/devel/community/communication.html?utm_medium=github&utm_source=issue_form--feature_request.yml#mailing-list-information
+
+      [new proposal]: ../../proposals/issues/new
 
 
 - type: textarea
   attributes:
     label: Summary
-    description: Describe the new feature/improvement briefly below.
+    description: >
+      Describe the new feature/improvement you would like briefly below.
+
+
+      What's the problem this feature will solve?
+
+      What are you trying to do, that you are unable to achieve
+      with ansible-core as it currently stands?
+
+
+      * Provide examples of real world use cases that this would enable
+      and how it solves the problem you described.
+      * How do you solve this now?
+      * Have you tried to workaround the problem using other tools?
+      * Could there be a different approach to solving this issue?
```

Because of `>` in YAML, we need more spacing
```suggestion
      * Provide examples of real-world use cases that this would enable
      and how it solves the problem you described.

      * How do you solve this now?

      * Have you tried to work around the problem using other tools?

      * Could there be a different approach to solving this issue?
```